### PR TITLE
Add search team roadmap items

### PR DESCRIPTION
**Pull request by lguychard**

### CODE REVIEW

File: handbook/product/roadmap.md [hunk 1]
```diff
@@ -74,7 +74,28 @@ See [Distribution roadmap](https://github.com/sourcegraph/about/pull/1104).
 
 ## Search
 
-TO DO
+1. 🔄 Enterprise homepage
```
Comment (nicksnyder) [inline handbook/product/roadmap.md:77]: How does this relate to and/or intersect with goals on the search team page?

File: handbook/product/roadmap.md [hunk 2]
```diff
@@ -74,7 +74,28 @@ See [Distribution roadmap](https://github.com/sourcegraph/about/pull/1104).
 
 ## Search
```
Comment (limitedmage) [inline handbook/product/roadmap.md:75]: Not sure if this belongs to the search team (it most likely belong to the web team I think), but there were talks a couple of weeks ago about reducing our cookie footprint to see if we can remove the cookie banner. I think this is important to track, so wanted to mention it here so we don't forget it.

File: handbook/product/roadmap.md [hunk 3]
```diff
@@ -74,7 +74,28 @@ See [Distribution roadmap](https://github.com/sourcegraph/about/pull/1104).
...
 
-TO DO
+1. 🔄 Enterprise homepage
+1. 🔄 Search tour
+1. 🔄 Search expressions
+1. 🔄 Scaling indexed text search to 500k repositories
+1. 🔄 Streaming search
+1. Code monitoring (private code monitors, no sharing)
```
Comment (limitedmage) [inline handbook/product/roadmap.md:82]: Does this include webhooks or only email for the initial release? Can we put webhooks as a separate item in this list if we think it doesn't need to exist in the initial release?

File: handbook/product/roadmap.md [hunk 4]
```diff
@@ -74,7 +74,28 @@ See [Distribution roadmap](https://github.com/sourcegraph/about/pull/1104).
...
+1. Performance at scale: follow-up to “scale indexed text search to 500k repositories”, focus and scope TBD
+1. As a user, I can create a scope of code I want to search across as a list of repositories (consolidate repogroups + search scopes)
```
Comment (limitedmage) [inline handbook/product/roadmap.md:84]: Do we want to call out the term "Graphs" here?

**Review by limitedmage — APPROVED** [review]

File: handbook/product/roadmap.md (hunk 1 above)
Comment (lguychard) [reply handbook/product/roadmap.md:77]: The in-progress items map directly to the [goals](https://about.sourcegraph.com/handbook/engineering/search#goals) listed on the search team page.

File: handbook/product/roadmap.md (hunk 3 above)
Comment (lguychard) [reply handbook/product/roadmap.md:82]: @poojaj-tech @christinaforney should we have a separate roadmap item here?

File: handbook/product/roadmap.md (hunk 4 above)
Comment (lguychard) [reply handbook/product/roadmap.md:84]: While it is possible that "graphs" will be the solution we go with to solve this problem, it is not certain yet, which is why we didn't call it out here.

File: handbook/product/roadmap.md (hunk 3 above)
Comment (lguychard) [reply handbook/product/roadmap.md:82]: Stage 1 of code monitoring as described in [code monitoring planning](https://docs.google.com/document/d/1EG42dM1old49uPXpGAqL7b9k95bWpM0Zcu0JM5gXtOQ/edit) includes webhooks, so let's just make them explicit in this roadmap item?

```suggestion
1. Code monitoring (private code monitors, no sharing, emails + webhooks)
```

Comment (limitedmage) [reply handbook/product/roadmap.md:82]: Sounds good!

File: handbook/product/roadmap.md (hunk 2 above)
Comment (poojaj-tech) [reply handbook/product/roadmap.md:75]: @limitedmage  thanks for calling this out. I’ll follow up with the web team (cc: @Joelkw) to figure out where this falls and the scope, and can update this accordingly.

**Review by poojaj-tech — APPROVED** [review]

File: handbook/product/roadmap.md (hunk 2 above)
Comment (poojaj-tech) [reply handbook/product/roadmap.md:75]: @limitedmage @Joelkw fyi opened this issue to track the cookie banner work (unassigned to team for now) [#14724](https://github.com/sourcegraph/sourcegraph/issues/14724)

**Comment by rvantonder** [comment]
Post merge 👍

**Pull request merged**